templates/al2/runtime/bootstrap.sh

```diff
@@ -465,13 +465,17 @@ if [[ -z "${DNS_CLUSTER_IP}" ]]; then
 
   if [[ "${IP_FAMILY}" == "ipv4" ]]; then
     if [[ ! -z "${SERVICE_IPV4_CIDR}" ]] && [[ "${SERVICE_IPV4_CIDR}" != "None" ]]; then
-      #Sets the DNS Cluster IP address that would be chosen from the serviceIpv4Cidr. (x.y.z.10)
+      # Sets the DNS Cluster IP address that would be chosen from the serviceIpv4Cidr. (x.y.z.10)
+      log "INFO: Detected SERVICE_IPV4_CIDR='$SERVICE_IPV4_CIDR'. Will set DNS_CLUSTER_IP as '${SERVICE_IPV4_CIDR%.*}.10'."
       DNS_CLUSTER_IP=${SERVICE_IPV4_CIDR%.*}.10
```

Comment:
If `--dns-cluster-ip` not set, it could at least resolve `DNS_CLUSTER_IP` from the result of `Line-382`

```
SERVICE_IPV4_CIDR=$(cat $DESCRIBE_CLUSTER_RESULT | awk '{print $6}')
```